gridcapa-core-cc-api/src/main/java/com/farao_community/farao/gridcapa_core_cc/api/util/IntervalUtil.java

```diff
@@ -20,23 +21,29 @@
  */
 public final class IntervalUtil {
     public static final ZoneId ZONE_ID = ZoneId.of("Europe/Brussels");
+    public static final int OFFSET_BEFORE_WINTER_DST = 2;
```

Comment:
Those two new constants are only used once to build a ZoneOffset.ofHours(xxx). Maybe the constant should be set to the ZoneOffset directly?